### ISSUE TITLE
Haiku: save last haiku, store haikus like quotes

### ISSUE THREAD

**Issue by richteer** [Halibot/haltoys#2]
What the title says
Sometimes things are poetic
So lets recall them

**Comment by sjrct** [comment]
I plan on someday
Rewriting some of haiku
Will implement then

I am hoping to get a better false positive/false negative rate with a rework of the core logic, will probably split the module into separate files then so it might merit a different repo. I have been meaning to do this for a while, I just haven't had the time to get around to it. If somebody else wants to do this, do let me hold you up, but I will definitely at some time in the future get around to this maybe.

**Comment by richteer** [comment]
I agree, Haiku should probably go in its own repo at this point. If you want to do that, go for it, otherwise I will migrate it when I have a more stable and trustworthy internet connection
